package.json

```diff
@@ -27,7 +27,8 @@
     "lint:post-build": "node dist/index.js lint src test --ignore-pattern 'test/tests/lint'",
     "test": "yarn build && yarn test:post-build",
     "test:post-build": "node dist/index.js test --config ./test/jest.config.json",
-    "start": "tsc -p tsconfig.json --watch"
+    "start": "tsc -p tsconfig.json --watch",
+    "publish": "np"
```

Comment:
`publish` is a lifecycle hook that's [actually equivalent to `postpublish`](https://docs.npmjs.com/misc/scripts). `np`'s docs say to use something like [`release`](https://github.com/sindresorhus/np#release-script). I've used `pub` in the past too.